### DISCUSSION TITLE
Webpack plugin

### DISCUSSION THREAD

**Discussion by panayotoff** [woltapp/blurhash#186] [Ideas]
Hi all,

What do you think about Webpack plugin that inlines the base64 image till the real image is loaded?

**Comment by DagAgren** [comment]
I heartily recommend implementing these things yourself and publishing them, and filing a PR to add a link to the README here. The code is very simple and easy to include anywhere you want or to port to whatever language you need!

**Comment by serushakov** [comment]
I wrote a POC loader for webpack, that would pre-process imported images and provide both blurhash and a url for the image, but it's a bit half baked. If there will be more requests I'll think about finalizing it and adding it here

**Comment by jtrein** [comment]
@serushakov This sounds great to me. We host some of our front-ends on Netlify and use Create React App. Having this as a part of the build process would be great!

**Comment by avaleriani** [comment]
@serushakov any news from this project? Is it open for contributions?
